### PR TITLE
Release the iotcentral sdk for the new sku name ST0, ST1, ST2

### PR DESCRIPTION
**Pull request by poanchen**
Swagger changes: https://github.com/Azure/azure-rest-api-specs/pull/7896
* Add new sku name ST0, ST1, ST2 for Microsoft.IoTCentral application.
* Update NuGet package version to 2.1.0 stable for the IoT Central.

### CODE REVIEW

File: sdk/iotcentral/Microsoft.Azure.Management.IotCentral/src/Microsoft.Azure.Management.IotCentral.csproj [hunk 1]
```diff
@@ -8,13 +8,11 @@
...
     <AssemblyName>Microsoft.Azure.Management.IotCentral</AssemblyName>
-    <Version>2.0.0</Version>
+    <Version>2.1.0</Version>
     <PackageTags>Microsoft Azure IotCentral;IotCentral management;IotCentral;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        This release adds a new API for checking the uniqueness of the application subdomain. 
-        Fixes the expected error object returned from Microsoft.IotCentral Rp service, for friendly messages.
-        Allows checking the uniqueness of different Microsoft.IotCentral resource types.
+        This release adds new sku name, ST0, ST1, ST2 for Microsoft.Azure.Management.IotCentral application.
```
Comment (emgarten) [inline sdk/iotcentral/Microsoft.Azure.Management.IotCentral/src/Microsoft.Azure.Management.IotCentral.csproj]: nit: This release adds new skus: ST0, ST1, ST2 for Microsoft.Azure.Management.IotCentral.

**Comment by poanchen** [comment]
Hey @erich-wang ,

Please review our PR and let us know what work we need to do in order to merge this PR.

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @poanchen , please
- update `assemblyfileversion` in `assemblyinfo.cs`
- write tests to cover new operations
Thanks

**Comment by kawayiboy** [comment]
> Hi @poanchen , please
> 
> * update `assemblyfileversion` in `assemblyinfo.cs`
> * write tests to cover new operations
>   Thanks

@isra-fel , the fix PR is here: https://github.com/Azure/azure-sdk-for-net/pull/9224, thanks

**Comment by isra-fel** [comment]
@kawayiboy @poanchen I've merged #9224 , so closing this one as well.

I also realized the folder structure of your module is wrong... at least it is different from other modules -- 

Should be: `sdk/iotcentral/Microsoft.Azure.Management.IotCentral/src/Generated`
Actually is: `sdk/iotcentral/Microsoft.Azure.Management.IotCentral/src/IotCentral/Microsoft.Azure.Management.IotCentral/src/Generated/`

It doesn't make any difference in the sense of the final delivery package, but fixing this can prevent confusing PRs such as this one in the future. I'll raise the PR and assign you to review.

**Comment by poanchen** [comment]
@isra-fel in order to change the folder structure, do I simply move it from sdk/iotcentral/Microsoft.Azure.Management.IotCentral/src/IotCentral/Microsoft.Azure.Management.IotCentral/src/Generated/
to
sdk/iotcentral/Microsoft.Azure.Management.IotCentral/src/Generated?

**Pull request closed without merging**